README.md

```diff
@@ -1,6 +1,6 @@
 # Eclipse Package Drone™
 
-Package Drone is a software artifact repository with an initial focus on OSGi.
+Package Drone is a software artifact repository with an initial focus on OSGi(Open Services Gateway initiative).
```

Comment:
```suggestion
Package Drone is a software artifact repository with an initial focus on OSGi (Open Services Gateway initiative).
```

Reply:
@ctron that has been done.

Reply:
I can't see that change yet. Maybe did you forget to push changes?